src/components/frame/v5/pages/DashboardPage/partials/Menu/Menu.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { FC } from 'react';
+import { usePopperTooltip } from 'react-popper-tooltip';
+import { useColonyContext } from '~hooks';
+import { formatText } from '~utils/intl';
+import Link from '~v5/shared/Link';
+import BurgerMenu from '~v5/shared/BurgerMenu';
+import PopoverBase from '~v5/shared/PopoverBase';
+import useColonySubscription from '~hooks/useColonySubscription';
+import { COLONY_DETAILS_ROUTE } from '~routes';
+import ShareIcon from '../ShareIcon';
+import MenuContentItem from '../MenuContentItem';
+
+const displayName = 'v5.pages.DashboardPage.partials.Menu';
+
+const Menu: FC = () => {
+  const { colony } = useColonyContext();
+  const { handleUnwatch } = useColonySubscription();
+
+  const {
+    getTooltipProps,
+    setTooltipRef,
+    setTriggerRef,
+    visible: isPopoverOpen,
+  } = usePopperTooltip({
+    delayShow: 200,
+    delayHide: 200,
+    placement: 'bottom-start',
+    trigger: ['click'],
+    interactive: true,
+  });
+
+  return (
+    <>
+      <BurgerMenu
+        isVertical
+        setTriggerRef={setTriggerRef}
+        className="text-gray-900 !p-0"
+      />
+      {isPopoverOpen && (
+        <PopoverBase
+          setTooltipRef={setTooltipRef}
+          tooltipProps={getTooltipProps}
+          withTooltipStyles={false}
+          cardProps={{
+            rounded: 's',
+            hasShadow: true,
+            className: 'py-4 px-2',
+          }}
+          classNames="w-full sm:max-w-[17.375rem]"
+        >
+          <Link
```

Comment:
Can you create a component with groups of links or buttons and then use it here?